### ISSUE TITLE
Support remote entities

### ISSUE THREAD

**Issue by felipecrs** [dummylabs/thewatchman#132]
Looks like all remote entities are always reported as unknown.

**Comment by dummylabs** [comment]
Could you share a snippet of a YAML file with the remote entity declaration so we can troubleshoot this issue? Thanks!

**Comment by felipecrs** [comment]
[Android TV Card](https://github.com/Nerwyn/android-tv-card):

```yaml
type: grid
cards:
  - type: horizontal-stack
    cards:
      - type: tile
        entity: media_player.tv_da_sala_samsung
        show_entity_picture: false
        vertical: false
        hide_state: false
        state_content:
          - state
          - last-changed
  - type: custom:android-tv-card
    remote_id: remote.tv_da_sala_samsung
    media_player_id: media_player.tv_da_sala_samsung
    autofill_entity_id: true
    custom_actions:
      up:
        tap_action:
          action: key
          key: KEY_UP
      down:
        tap_action:
          action: key
          key: KEY_DOWN
      left:
        tap_action:
          action: key
          key: KEY_LEFT
      right:
        tap_action:
          action: key
          key: KEY_RIGHT
      center:
        tap_action:
          action: key
          key: KEY_ENTER
        double_tap_action:
          action: key
          key: KEY_RETURN
      power:
        icon: mdi:power
        tap_action:
          action: call-service
          service: media_player.toggle
      home:
        tap_action:
          action: key
          key: KEY_HOME
      back:
        tap_action:
          action: key
          key: KEY_RETURN
      volume_mute:
        tap_action:
          action: key
          key: KEY_MUTE
      input:
        icon: mdi:hdmi-port
        tap_action:
          action: key
          key: KEY_SOURCE
    rows:
      - - back
        - power
        - input
        - home
      - - volume_mute
        - slider
      - - touchpad
title: TV da sala (Samsung)
```

But this can also be present in automations and everywhere else AFAIK.

**Comment by RoboMagus** [comment]
I believe `remote` entities are a relatively new addition to the supported platforms.

It might be a good addition to have a GH workflow verify the platforms that Watchman checks against the [complete supported list](https://github.com/home-assistant/core/blob/3db4d951bfb237063d9846accf15057a51dda3de/homeassistant/const.py#L40) as part of a test case.

This could help keep track of any new additions, as well as provide opportunity to prune deprecated entity platforms such as `proximity`.

**Comment by dummylabs** [comment]
Will be fixed in 0.6.4
